libraries/core_libs/consensus/src/dag/sortition_params_manager.cpp

```diff
@@ -1,8 +1,11 @@
 #include "dag/sortition_params_manager.hpp"
 
 #include "pbft/pbft_block.hpp"
+
 namespace taraxa {
 
+constexpr uint16_t k_testnet_hardfork2_sortition_interval_length = 200;
```

Comment:
I am pretty confused and the wording seems weird. 

- This constant is basically computation_interval (200) and the original computation interval should be renamed to something like changing_interval(50) ? 
- Why is this not part of config ? If it was part of config, you would not need to use hardfork2 if conditions [here](https://github.com/Taraxa-project/taraxa-node/pull/1363/files#diff-740ffa012af5a6c39db38b0cba5ee86455365dd3397a774838fd38b043e5a8c6R157) and [here](https://github.com/Taraxa-project/taraxa-node/pull/1363/files#diff-740ffa012af5a6c39db38b0cba5ee86455365dd3397a774838fd38b043e5a8c6R80)

Reply:
Changing the config structure(introducing a new field, or even change a name) will create and additional task how to deal with that config change for node runners. 
For the develop branch it will be sure in the config, but for the testnet with a hardfork I am not sure we should spend time on that

Reply:
well we are getting into the situation where most of the lateest changes in testnet are like "lets merge this and do it properly in develop", but does anybody even keep track of it ? It will be pretty big mess if we continue do it like this. If authors keep track of it I guess it might be ok but I am not sure if it is the case